iOS Inspector_files/inspector.js

```diff
@@ -174,11 +174,11 @@ Inspector.prototype.initSessionId = function () {
         .done(function (data) {
                   me.log.info("success");
                   me.log.info(data);
-                  if (data.value[0] === undefined) {
+                  if (data.value[data.value.length - 1] === undefined) {
```

Comment:
`if (Array.isArray(data.value) && data.value.length)`

Reply:
I revised it with the suggestion above. Is that all right?

Reply:
not really. it would be enough to simply replace the first if condition with the proposed one to make sure the data.value is an array with at least one item, because reading `.length` property from it would throw an exception otherwise

Reply:
Well, I mean, if there are multiple sessions, the last session should be valid in most cases, and the first session will always be used in the case of value [0]. If the first session fails, the elements of the device will not be refreshed on the page. I'm not concerned about exception throw, but the above situation exists when using it.

Reply:
Very good. Thank you.